index.js

```diff
@@ -33,14 +34,8 @@ module.exports = function(options) {
     requireFn = function (name) {
       // This searches up from the specified package.json file, making sure
       // the config option behaves as expected. See issue #56.
-      var searchFor = path.join('node_modules', name);
-
-      var src = findup(searchFor, {cwd: path.dirname(config)}) || name;
-      if (src !== null) {
-        return require(src);
-      } else {
-        throw new Error('Cannot find `' + name + '` in your node_modules!');
-      }
```

Comment:
@callumacrae have we lost this nice error now? I'd be keen to keep that (if you fancy wrapping a test around it too...)